### ISSUE TITLE
[BITV] 9.1.3.1e/0.1 - Although the table is properly structured, the content composition of the "Name" column is incorrect. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36920]
Nested links have been used, which results in both the name of the file/directory and link text such as "Share" and "Actions" being read out together. The two buttons "Share" and "Actions" should be placed in their own table column. 

![796c868a23d4b0de118f5260f447c473](https://user-images.githubusercontent.com/1315170/221934554-4c1c8e12-11fc-44a8-929f-532269f54966.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-3f0ff0bc0e-v0-n1

</details>

**Comment by michaelnissenbaum** [comment]
Hi @JuliaKirschenheuter it looks good to me. I created a code example and tested it using VoiceOver/Safari and VoiceOver/Chrome. Both worked fine to me.

**Comment by AndyScherzinger** [comment]
fixed via f2v
